packages/infra/lib/lambdas-nested-stack.ts

```diff
@@ -0,0 +1,353 @@
+    super(scope, id, props);
+
+    this.lambdaLayers = setupLambdasLayers(this);
+
+    this.cdaToVisualizationLambda = this.setupCdaToVisualization({
+      lambdaLayers: this.lambdaLayers,
+      vpc: props.vpc,
+      envType: props.config.environmentType,
+      medicalDocumentsBucket: props.medicalDocumentsBucket,
+      sandboxSeedDataBucket: props.sandboxSeedDataBucket,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: props.alarmAction,
+    });
+
+    this.documentDownloaderLambda = this.setupDocumentDownloader({
+      lambdaLayers: this.lambdaLayers,
+      vpc: props.vpc,
+      secrets: props.secrets,
+      cwOrgCertificate: props.config.cwSecretNames.CW_ORG_CERTIFICATE,
+      cwOrgPrivateKey: props.config.cwSecretNames.CW_ORG_PRIVATE_KEY,
+      bucketName: props.medicalDocumentsBucket.bucketName,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+    });
+
+    this.outboundPatientDiscoveryLambda = this.setupOutboundPatientDiscovery({
+      lambdaLayers: this.lambdaLayers,
+      vpc: props.vpc,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: props.alarmAction,
+      dbCluster: props.dbCluster,
+      dbCredsSecret: props.dbCredsSecret,
+      // TODO move this to a config
+      maxPollingDuration: Duration.minutes(11),
+    });
+
+    this.outboundDocumentQueryLambda = this.setupOutboundDocumentQuery({
+      lambdaLayers: this.lambdaLayers,
+      vpc: props.vpc,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: props.alarmAction,
+      dbCluster: props.dbCluster,
+      dbCredsSecret: props.dbCredsSecret,
+      // TODO move this to a config
+      maxPollingDuration: Duration.minutes(15),
+    });
+
+    this.outboundDocumentRetrievalLambda = this.setupOutboundDocumentRetrieval({
+      lambdaLayers: this.lambdaLayers,
+      vpc: props.vpc,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: props.alarmAction,
+      dbCluster: props.dbCluster,
+      dbCredsSecret: props.dbCredsSecret,
+      // TODO move this to a config
+      maxPollingDuration: Duration.minutes(15),
+    });
+  }
+
+  private setupCdaToVisualization(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    envType: EnvType;
+    medicalDocumentsBucket: s3.Bucket;
+    sandboxSeedDataBucket: s3.IBucket | undefined;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const {
+      lambdaLayers,
+      vpc,
+      sentryDsn,
+      envType,
+      alarmAction,
+      medicalDocumentsBucket,
+      sandboxSeedDataBucket,
+    } = ownProps;
+
+    const cdaToVisualizationLambda = createLambda({
+      stack: this,
+      name: "CdaToVisualization",
+      nameSuffix: "v2",
```

Comment:
All lambdas being created on this PR have this as a way to name them differently than the original name.